include/posix/time.h

```diff
@@ -101,4 +106,9 @@ int nanosleep(const struct timespec *rqtp, struct timespec *rmtp);
 #include <syscalls/time.h>
 #endif /* CONFIG_ARCH_POSIX */
 
+#else /* ZEPHYR_INCLUDE_POSIX_TIME_H_ */
+/* Read the standard header when <posix/time.h> finds itself on the
```

Comment:
What "standard header"? If you mean "toolchain-provided header", as the commit message says, please say it so here too.